benchmarks/Microsoft.AspNetCore.Routing.Performance/Matching/JumpTableSingleEntryBenchmark.cs

```diff
@@ -40,21 +50,30 @@ public int Baseline()
             var strings = _strings;
             var segments = _segments;
 
-            var destination = 0;
-            for (var i = 0; i < strings.Length; i++)
+            int destination = 0;
+            for (int i = 0; i < strings.Length; i++)
```

Comment:
Ugh, the new VS 'code cleanup' feature did this. I'll never trust VS to do anything again.